docs/decision_record.md

```diff
@@ -6,6 +6,24 @@
 
 [TOC]
 
+## 2022-05-11 - Sequence identifier specification
+
+### Decision
+
+The GA4GH identifier will be used as our default sequence identifier instead of MD5. Other identifiers can be provided in a separate array and should not be part of the collection checksum calculation.
```

Comment:
I think we should specify where we intend to use the identifiers.  Something along the line of
```suggestion
The GA4GH identifier will be used as our default sequence identifier instead of MD5. Other identifiers can be provided in a separate array and should not be part of the collection checksum calculation.
It will be used to digest:

- the sequences that are stored in the `sequences` array
- the canonical representation of arrays of level 2
- the canonical representation of the sequence collection of level 1
```